### PR TITLE
DGJ-570 SL Review workflow updated, Triggers Camunda on application edit, flatten data on send to ODS

### PR DESCRIPTION
**Pull request by iman-jamali-fw**
## Summary

This PR (Ticker #570) enables the SL Review resubmission through the `application` tab and sends its data to ODS on each update.

## Changes
- SL Review workflow upated
- Logic to trigger camunda workflow under application edit was added
- The confirmation page for SL Review submit/resubmit was added

## Notes
Please note that the sendToODS listener in Camunda still needs some work including flatMap form data.

### CODE REVIEW

File: forms-flow-web/src/components/Form/Item/Submission/Item/Edit.js [hunk 1]
```diff
@@ -146,6 +177,34 @@ const Edit = React.memo((props) => {
     return <Loading />;
   }
 
+  const onApplicationFormSubmit = (actionType = "") => {
```
Comment (simensma-fresh) [inline forms-flow-web/src/components/Form/Item/Submission/Item/Edit.js:180]: I'm wondering if we can change this to be decoupled from the form "submit" button and instead be triggered through a Form Event to match how this happens on the Task page (ServiceTaskDetails.js -> onCustomEventCallBack). That way we can easily disable / enable this per form as needed and also have the option to pass along custom data to the Camunda Task you want to progress. You can check the "Submit Decision" button on the Telework form for that logic

```
  form.emit('customEvent', {
            type: "actionComplete",   
            component: component,
            actionType:data.action
      }); 
```

**Review by simensma-fresh — CHANGES_REQUESTED** [review]
Overall looks good! Two suggestions on how to make this a bit more reusable and flexible

File: forms-flow-web/src/components/Form/Item/Submission/Item/Edit.js (hunk 1 above)
Comment (simensma-fresh) [reply forms-flow-web/src/components/Form/Item/Submission/Item/Edit.js:180]: And if you're able to add the ticket number to the title of the PR, that would be great! Just makes it easier to reference this if needed in the future

File: forms-flow-bpm/src/main/java/org/camunda/bpm/extension/hooks/listeners/SendSubmissionToODSDelegate.java [hunk 2]
```diff
@@ -53,22 +55,23 @@ private void sendSubmissionToODS(DelegateExecution execution) throws IOException
         Object managerIdir = execution.getVariable("manager_idir");
         Object managerGuid = execution.getVariable("manager_guid");
 
+        // Todo: Replace the form name with an env
+        boolean hasNestedObjects = formName.equals("SLReview");
```
[the remark this answers is not in the document]
Comment (iman-jamali-fw) [reply forms-flow-bpm/src/main/java/org/camunda/bpm/extension/hooks/listeners/SendSubmissionToODSDelegate.java]: addresed

File: forms-flow-web/src/components/Form/Item/Submission/Item/Edit.js (hunk 1 above)
Comment (iman-jamali-fw) [reply forms-flow-web/src/components/Form/Item/Submission/Item/Edit.js:181]: addressed.

**Pull request merged**